build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -454,11 +494,15 @@ String _requireJsConfig(bool soundNullSafety) => '''
 
 $_baseUrlScript;
 
-require.config({
-    baseUrl: baseUrl,
-    waitSeconds: 0,
-    paths: customModulePaths
-});
+if (!window.\$dartRequire) {
+  window.\$dartRequire = new Map();
+}
+window.\$dartRequire.set("$contextId", require.config({
+  baseUrl: baseUrl,
+  context: "$contextId",
+  waitSeconds: 0,
+  paths: customModulePaths
+}));
```

Comment:
Is this what you were thinking for namespacing these custom requirejs contexts?

Reply:
Yes 👍